### ISSUE TITLE
Merge with custom_chewie plugin

### ISSUE THREAD

**Issue by cbenhagen** [brianegan/chewie#105]
Let's merge back the features of the [custom_chewie](https://pub.dartlang.org/packages/custom_chewie) plugin. Unless there are very good technical reasons I don't see the point of having two very similar plugins that even have almost the same name.

We recently did a major refactor of the code which should make it much easier to customise things. https://github.com/brianegan/chewie/issues/100 tracks the effort to make customising more end-user friendly.

@bostrot, what do you think? Would you want to collaborate on this?

**Comment by bostrot** [comment]
Thats true. There isnt even a point in the other plugin. Unfortunately I don't have much time at the moment for a rewriting for a pull request.
Though I would really like to cooperate on this one.

**Comment by cbenhagen** [comment]
Guess you could start by creating issues for features that need to be merged. With a quick description and if you have, some information about things to keep in mind when implementing. I can then try to do them and ping you if I need assistance. As soon as those are all fixed we can add a notice to the README of your fork and push that to pub. Deleting from pub is not possible as far as I know.
